colonist.io

SKEIDs opened the pull request:
#104316

SKEIDs committed:
remove

SKEIDs committed:
Add

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Add

SKEIDs committed:
Update specific.txt

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'master' of https://github.com/SKEIDs/AdguardFilters

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
Merge branch 'master' of https://github.com/SKEIDs/AdguardFilters

SKEIDs committed:
Merge branch 'AdguardTeam:master' into master

SKEIDs committed:
fix #104316 colonist.io

Yuki2718 reviewed (changes requested):
Yours is okay, but I opened PR in EasyList so if they're quick to merge you may need to remove these rules (no need to do currently). Until then please replace `||colonist.io/dist/images/binance*x*.jpg` with `||colonist.io/dist/images/ab_`.

SKEIDs committed:
rule was replace

https://github.com/AdguardTeam/AdguardFilters/pull/104329#pullrequestreview-839975137

SKEIDs commented:
@Yuki2718 
OK.Done.
